### ISSUE TITLE
Implementation for .txt files

### ISSUE THREAD

**Issue by simalesu** [AFM-analysis/afmformats#20]

**Comment by paulmueller** [comment]
@simalesu May I attach the files you sent me via email to this issue? Do you have any problems making those files available under a pubic domain license (CC-0)? I need this so I can incorporate them into the testing framework.

Could you also please add more information about the measurements? Things such as spring constant. Maybe a screenshot from another analysis software. Something to test against to make sure the files are imported correctly.

**Comment by simalesu** [comment]
Yes, you can attach these files. My advisor gave me the files actually for just testing. He suggested to give the spring constant and sensitivity arbitrary. Could it possible?

**Comment by simalesu** [comment]
I don't know actually which parameters do I have for these files but we can insert the parameters arbitrarily such as k= 0.1 N/m and 10 N/m. It will be only for testing.

**Comment by simalesu** [comment]
I don't know actually which parameters do I have for these files but we can 
insert the parameters arbitrarily such as k= 0.1 N/m and 10 N/m. It will be 
only for testing.

Am Mon, 31 Jan 2022 02:01:12 -0800 schrieb Paul Müller 
***@***.***>:
